plugins/language_htaccess.lua

```diff
@@ -0,0 +1,76 @@
+-- mod-version:2 -- lite-xl 2.0
+local syntax = require "core.syntax"
+
+local keywords = {
+  "AddCharset", "AddDefaultCharset", "AddHandler", "AddOutputFilterByType", "AddType", "Allow",
+  "AuthName", "AuthType", "AuthUserFile", "BrowserMatch", "CheckSpelling", "DefaultLanguage",
+  "Deny", "DirectoryIndex", "ErrorDocument", "ExpiresActive", "ExpiresByType", "ExpiresDefault",
+  "FallbackResource", "FileETag", "ForceType", "from", "Header", "IndexIgnore", "LimitRequestBody",
+  "LoadModule", "Options", "Order", "php_flag", "php_value", "Redirect", "RedirectMatch",
+  "RequestHeader", "Require", "RewriteBase", "RewriteCond", "RewriteEngine", "RewriteRule",
+  "Satisfy", "ServerSignature", "SetEnv", "SetEnvIf", "SetEnvIfNoCase",
+}
+local literals = {
+  "on", "off", "deny", "denied", "all", "All", "allow", "Basic", "valid-user", "append", "unset",
+  "set", "DEFLATE",
+}
+
+local symbols = {}
+for _,kw in ipairs(keywords) do
+  symbols[kw] = "keyword"
+end
+for _,lt in ipairs(literals) do
+  symbols[lt] = "literal"
+end
+
+local url_syntax = {
+  patterns = {
+    { pattern = "[%%$]%{?[%w_]+%}?", type = "keyword2" },
+    { pattern = "[^%%$%s]", type = "string" }
+  },
+  symbols = {}
+}
+local xml_syntax = {
+  patterns = {{ pattern = { '"', '"', '\\' }, type = "string" }},
+  symbols = {}
+}
+
+syntax.add {
+  name = ".htaccess File",
+  files = { "%.htaccess$" },
```

Comment:
Should this match `.htaccess` only, or can those files begin with something else (`example.htaccess`)?

If the former, I'd use:
```suggestion
  files = { "^%.htaccess$" },
```
If the latter, then it's correct as it is now.

Reply:
Fixed as of latest commit.